s3-import-commons/src/test/java/no/unit/nva/publication/s3imports/FileEntriesEventEmitterTest.java

```diff
@@ -123,6 +126,16 @@ public void handlerEmitsEventWithResourceWhenFileUriExistsAndContainsDataAsJsonO
         assertThat(emittedResourceObjects, containsInAnyOrder(FILE_01_CONTENTS));
     }
 
+    @Test
+    public void handlerEmitsEventWithTimestampInformation() {
```

Comment:
not accurate enough. this description can mean anything. The test name is 
`handlerEmitsEventWithTimestampAcquiredByImportRequest`.
If you do not specify the exact functionality in the title, the next person will have no idea what this test does without looking through the code. It is essential that one understands the implemented functionality by just reading the test names. That way one can refactor the code being sure that they do not break functionality.

Our job is basically to define functionality (currently through tests) and unfortunately the technology is not that advanced and so we have to implement the functionality as well. But the core of our job is to define the functionality well.